File: aspnet/web-forms/overview/older-versions-security/membership/user-based-authorization-cs/samples/sample21.cs
// Is this Tito visiting the page?
string userName = User.Identity.Name;
if (string.Compare(userName, "Tito", true) == 0)
	// This is Tito, SHOW the Delete column
	FilesGrid.Columns[1].Visible = true;
else
	// This is NOT Tito, HIDE the Delete column
	FilesGrid.Columns[1].Visible = false;

File: aspnet/web-forms/overview/older-versions-security/membership/user-based-authorization-vb/samples/sample21.vb
' Is this Tito visiting the page?
Dim userName As String = User.Identity.Name
If String.Compare(userName, "Tito", True) = 0 Then
 ' This is Tito, SHOW the Delete column
 FilesGrid.Columns(1).Visible = True
Else
 ' This is NOT Tito, HIDE the Delete column
 FilesGrid.Columns(1).Visible = False
End If